tempesta_fw/addr.c

```diff
@@ -233,28 +232,29 @@ tfw_addr_pton(const TfwStr *str, TfwAddr *addr)
 	int ret = -EINVAL;
 	int mode = 0;
 	const char first = TFW_STR_PLAIN(str) ?
-		*(char*)str->ptr :
-		*(char*)((TfwStr*)str->ptr)->ptr;
+	*str->data :
+	*(str->chunks)->data;
```

Comment:
Wrong formatting. The correct way of writing a code like this is would be something like the following:
```C
        const char first = TFW_STR_PLAIN(str)
                           ? *str->data : *(str->chunks)->data;
```

or, maybe like this, but I don't see a reason to introduce an extra line here:
```C
        const char first = TFW_STR_PLAIN(str)
                           ? *str->data
                           : *(str->chunks)->data;
```